components/select/apiExamples/labelWithPlaceholder.html

```diff
@@ -1,5 +1,6 @@
-<auro-select placeholder="Placeholder Text">
+<auro-select>
   <span slot="label">Label Text</span>
+  <label slot="placeholder">Placeholder Text</label>
```

Comment:
Add placeholder to all examples to demonstrate that it should always be used by implementers